### PR TITLE
Outline the purpose of the repository and generate project metadata

### PR DESCRIPTION
**Pull request by taylorreiter**
This pull request starts this repository. It does two things:

1. starts an aspirational README file. The README describes the purpose of the repository and what the code will eventually do. It also outlines sections that will be filled in as the code is written and run.
2. programmatically generates a metadata file that will be ingested by the pipeline to accomplish the repository's goals.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,19 @@
-# 2022-mtx-not-in-mgx-pairs
+# Identifying sequences that are in a metatranscriptome but not in a metagenome
+
+This repository curates a set of paired metagenomes and metatranscriptomes and provides a pipeline to rapidly identify the fraction of sequences in a metatranscriptome that are not in a metagenome.
+The pipeline is shaped around a metadata file, 'inputs/metadata-paired-mgx-mtx.tsv', that contains a sample name (`sample_name`), metagenome SRA run accession (`mgx_run_accession`; `SRR*`, `ERR*`, `DRR*`), metatranscriptome SRA run accession (`mtxx_run_accession`), and a sample type (`sample_type`).
```
Comment (borgesadair1) [inline README.md]:
```suggestion
The pipeline is shaped around a metadata file, 'inputs/metadata-paired-mgx-mtx.tsv', that contains a sample name (`sample_name`), metagenome SRA run accession (`mgx_run_accession`; `SRR*`, `ERR*`, `DRR*`), metatranscriptome SRA run accession (`mtx_run_accession`), and a sample type (`sample_type`).
```

**Review by borgesadair1 — APPROVED** [review]
Looks good to me, other than the `mtxx` typo which i hope to have corrected in line.

File: inputs/metadata-parse-paired-mgx-mtx.ipynb [hunk 2]
```diff
@@ -0,0 +1,5079 @@
+{
```
Comment (borgesadair1) [inline inputs/metadata-parse-paired-mgx-mtx.ipynb:1]: <p>Typo in header : availabl --&gt; available </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arcadia-Science/2022-mtx-not-in-mgx-pairs/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (borgesadair1) [inline inputs/metadata-parse-paired-mgx-mtx.ipynb:1]: <p>mRNA enriched would be preferable here, total RNA will be largely ribosomal. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arcadia-Science/2022-mtx-not-in-mgx-pairs/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (borgesadair1) [inline inputs/metadata-parse-paired-mgx-mtx.ipynb:1]: <p>Can we combine <code>human_gut</code> and <code>human_gut_microbiome</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arcadia-Science/2022-mtx-not-in-mgx-pairs/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='97'/>

Comment (taylorreiter) [reply inputs/metadata-parse-paired-mgx-mtx.ipynb:1]: <p>good catch! I actually only used mRNA-enriched per our slack conversation, I'll update this comment to reflect that!</p>

Comment (taylorreiter) [reply inputs/metadata-parse-paired-mgx-mtx.ipynb:1]: <p>yes great catch, thank you!</p>

Comment (taylorreiter) [reply inputs/metadata-parse-paired-mgx-mtx.ipynb:1]: I also removed `_microbiome` from `snail_gut` bc they're all microbiomes and that was the only one that still had it in it.

**Pull request merged**